api/setup.py

```diff
@@ -45,13 +45,22 @@ def get_requirements(req_file):
 packages = find_packages("src", include=["backend_common", "cli_common"]) + find_namespace_packages("src", include=["shipit_api.common", "shipit_api.public"])
 name = "shipit_api-public"
 req_file = "public.txt"
+entry_points = None
 
 # In order to separate the public and admin apps, use `APP_TYPE` environment
 # variable as a most portable way to parameterize setup.py
 if os.environ.get("APP_TYPE") == "admin":
     packages.extend(find_namespace_packages("src", include=["shipit_api.admin"]))
     name = "shipit_api-admin"
     req_file = "base.txt"
+    entry_points = {
+        "console_scripts": [
+            "shipit_upload_product_details = shipit_api.admin.cli:upload_product_details",
```

Comment:
Nit: 

```suggestion
            "shipit_upload_product_details = shipit_api.admin.cli:download_product_details",
```

https://github.com/mozilla-releng/shipit/blob/db32cee78544d13847269c45f39ed7281e837b5a/api/src/shipit_api/admin/cli.py#L61-L65

Reply:
Whoops, good catch!